interceptors/1.2/_index.md

```diff
@@ -33,4 +33,4 @@ and other managed classes.
 
 # Compatible Implementations
 
-* [Eclipse GlassFish 5.1](https://eclipse-ee4j.github.io/glassfish/)
\ No newline at end of file
+* [Eclipse GlassFish 5.1](https://www.eclipse.org/downloads/download.php?file=/glassfish/glassfish-5.1.0.zip)
```

Comment:
Based on a mailing list conversation, I believe we decided that we didn't like the direct link to the glassfish download.  Instead we should point at either the downloads page (https://projects.eclipse.org/projects/ee4j.glassfish/downloads) or a github download page (https://eclipse-ee4j.github.io/glassfish/download) or the project page...

Reply:
I changed this to be aligned with the ticket: 

https://github.com/eclipse-ee4j/jakartaee-platform/issues/99

There it's the direct download. I'll make it the project download page for now then. Thx!